tezos/api/src/ffi.rs

```diff
@@ -751,6 +752,58 @@ pub struct RpcRequest {
     pub accept: Option<String>,
 }
 
+fn ffi_routed_subpath(path: &str) -> String {
```

Comment:
Btw, I don't like this name, suggestions welcome. Maybe `ffi_rpc_router_cache_key` or something like that, dunno.

Reply:
@tizoc hehe, just add please some docs/comments for function

Reply:
It is in `RpcRequest::ffi_routed_subpath` this one here was just extracted out so that the tests don't need to construct the whole `RpcRequest` just to test this. Btw I am going to rename it to `ffi_rpc_router_cache_key` to make it very explicit that it is why this function exists.